### PR TITLE
UG Update with Known Issues

### PR DESCRIPTION
**Pull request by DennieDan**
Minor grammar change with Known Issues related to 2dp differences

### CODE REVIEW

**Review by teamotitan — APPROVED** [review]
LGTM

**Review by Feng1231 — APPROVED** [review]

**Review by TY1Fan — APPROVED** [review]
lgtm

File: docs/UserGuide.md [hunk 1]
```diff
@@ -125,14 +125,16 @@ Action     | Format, Examples
...
 
+<div style="page-break-after: always;"></div>
 
 ### Viewing help : `help`
 
 Shows a message explaining how to access the help page.
 
+**Format**: `help`
+
 ![help message](images/helpMessage.png)
```
Comment (TY1Fan) [inline docs/UserGuide.md:136]: Hi can you add a **Output:** before this line

**Review by Soumyadip-cmd — APPROVED** [review]
LGTM

**Pull request merged**